code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -44,13 +44,25 @@ commandLine = text $ "In your terminal command line, enter the same directory as
 -- | Helper for giving instructions on how to run the program.
 runInstDoc :: Doc
 runInstDoc = text "How to Run the Program:" <> contSep <>
-    commandLine <> contSep <> bkQuote3 <> contSep <> text "make run RUNARGS=input.txt" <> contSep <> bkQuote3
+    commandLine <> contSep <> bkQuote3 <> contSep <> text "make run RUNARGS=input.txt" 
+      <> contSep <> bkQuote3
 
 -- | Helper for giving instructions on how to build the program.
 makeInstDoc :: Doc
 makeInstDoc = text "How to Build the Program:" <> contSep <> commandLine <> contSep <>
     bkQuote3 <> contSep <> text "make build" <> contSep <> bkQuote3
 
+-- | Helper for giving instuctions and Input and Output files.
+inOutFile :: String -> Maybe String -> Maybe String -> Doc
+inOutFile _ Nothing _ = empty
+inOutFile _ _ Nothing = empty
+inOutFile name (Just inFile) (Just outFile) = doubleSep <>
+      text "How to Change Input:" <> contSep <> text name <+> 
+      text "will take the inputs from" <+> bkQuote <> text inFile <> bkQuote <+> 
+      text "and write the outputs to" <+> bkQuote <> text outFile <> bkQuote <> 
+      text "." <> contSep <> text "Inputs can be changed by editing" <+> bkQuote <> 
+      text inFile <> bkQuote <> text "."
```

Comment:
Maybe add a comment to this function saying that it is to be expanded upon in the future, similar to the `-- This needs a more permanent solution` comment above.